packages/node_modules/@ciscospark/widget-meet/src/messages.js

```diff
@@ -34,6 +50,10 @@ export default defineMessages({
     id: `ciscospark.container.meet.message.noWebRTCBrowserSupport`,
     defaultMessage: `Video calling is not yet supported in your browser.`
   },
+  unableToCallMessage: {
+    id: `ciscospark.container.meet.message.unableToCallMessage`,
+    defaultMessage: `Unable to connect call`
```

Comment:
Unable to call is the main error message while the others are the subtext. Probably don't need "Cisco Spark" in both.